content/projects/tdd/string-calculator/_index.md

```diff
@@ -186,7 +200,7 @@ Hint: A valid string input follows these formats:
 - "//[delimiter][delimiter]\n integer delimiter integer" e.g "//[\*][%]\n1\*2%3"
 ```
 
-If the string does not abide by any of these formats, it should be considered invalid.
+If the string does not abide by any of these formats, it should be considered invalid. Square brackets (`[ or ]`)are used as identifiers, and will not be used as delimiters. Any string with these as delimiters should also be considered invalid.
```

Comment:
What are these that you are referencing to?

Reply:
What I mean here is the square brackets `[` or `]`.